### DISCUSSION TITLE
Mapbox customized Japanese map.

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#1165] [Q&A] [answered]
Kansai Website team in Japan made customized Japanese UI Mapbox's map. Their map shows up more Japanese, street name and more. And Mapbox provided Style URL. 

My question is:
Is it possible to use their map? and if so how they can put Mapbox API with Style URL on their website?

If you have any questions, please let me know.

**Comment by joshreisner** [comment]
Hi Hiro, how does the customized map work? Is there a particular embed code?

**Reply by Hiro-Tokyo** [reply]
Thanks Josh!

Here's some info. They replaced access token but it did not work well. 

They made their map following this instruction.

https://docs.mapbox.com/studio-manual/guides/publish-your-style/

Here's their map's URL

https://api.mapbox.com/styles/v1/aa-kansai-hp/cljrn68ey006401rg3n3j9wdk.html?title=false&access_token=pk.eyJ1IjoiYWEta2Fuc2FpLWhwIiwiYSI6ImNsZXRkcTZ5ZTAxbzczdm85emZrY251cWgifQ.tkZxwY6cZIMVSJ-IJkrD4g&zoomwheel=false#14.14/34.77256/135.46966

and style URL is 

mapbox://styles/aa-kansai-hp/cljrn68ey006401rg3n3j9wdk

**Reply by Hiro-Tokyo** [reply]
And they tried this access token.

pk.eyJ1IjoiYWEta2Fuc2FpLWhwIiwiYSI6ImNsZXRkcTZ5ZTAxbzczdm85emZrY251cWgifQ.tkZxwY6cZIMVSJ-IJkrD4g

**Reply by joshreisner** [reply, marked as answer]
ah ok! you should be able to provide your map style in the settings object like this (keep whatever other settings you have in there and just add `map.style`):

```js
var tsml_react_config = {
  distance_unit: 'km',
  map: {
    style: 'mapbox://styles/aa-kansai-hp/cljrn68ey006401rg3n3j9wdk',
  },
};
```

**Reply by Hiro-Tokyo** [reply]
Thanks Josh! We will try it!

**Reply by Hiro-Tokyo** [reply]
It works well! Thanks!
